src/components/StdinContext.ts

```diff
@@ -0,0 +1,26 @@
+import {createContext} from 'react';
+
+export interface StdinProps {
```

Comment:
```suggestion
export interface StdinContextProps {
```

Reply:
Changed in https://github.com/vadimdemedes/ink/pull/264/commits/d09a2f3de338bc49064c467827ba17a04087b834